src/main/java/com/cburch/logisim/vhdl/sim/VhdlSimulatorVhdlTop.java

```diff
@@ -78,57 +78,53 @@ public void generate(List<Component> comps) {
     /* Do not generate if file is already valid */
     if (valid) return;
 
-    StringBuilder ports = new StringBuilder();
-    ports.append("Autogenerated by logisim --");
+    final var ports = new StringBuilder();
+    ports.append(StringUtil.format("Autogenerated by %s --", Main.APP_DISPLAY_NAME));
     ports.append(System.getProperty("line.separator"));
 
-    StringBuilder components = new StringBuilder();
+    final var components = new StringBuilder();
     components.append("Autogenerated by logisim --");
     components.append(System.getProperty("line.separator"));
 
-    StringBuilder map = new StringBuilder();
+    final var map = new StringBuilder();
     map.append("Autogenerated by logisim --");
```

Comment:
Here would be another candidate place to use `APP_DISPLAY_NAME`.

Reply:
Blind me. Fixed